### PR TITLE
Wingware.WingIDE101 version 9.0.0.4

### PR DESCRIPTION
**Pull request by danielsherratt**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/87330)

### CODE REVIEW

**Comment by danielsherratt** [comment]
@microsoft-github-policy-service agree

File: manifests/w/Wingware/WingIDE101/9.0.0.4/Wingware.WingIDE101.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Wingware.WingIDE101
+PackageVersion: 9.0.0.4
+PackageLocale: en-US
+Publisher: Wingware
+PackageName: WingIDE 101
+License: Wingware
+Copyright: Copyright © 1999-2022 Archaeopteryx Software, Inc.
+ShortDescription: Wing IDE 101 is a free tool to learn programming
```
Comment (danielsherratt) [inline manifests/w/Wingware/WingIDE101/9.0.0.4/Wingware.WingIDE101.locale.en-US.yaml:11]: ShortDescription: Wing 101 is the simplest IDE to teach students Python

**Review by danielsherratt — COMMENTED** [review]
Can the ShortDescription be changed to **Wing 101 is the simplest IDE to teach students Python**

File: manifests/w/Wingware/WingIDE101/9.0.0.4/Wingware.WingIDE101.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Wingware.WingIDE101
+PackageVersion: 9.0.0.4
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://wingware.com/pub/wing-101/9.0.0.4/wing-101-9.0.0.4.exe
+  InstallerSha256: 2875732AF1CBFE98E65E625215F82275E825DA001B50EF3F28620F4217A376FF
```
Comment (Trenly) [inline manifests/w/Wingware/WingIDE101/9.0.0.4/Wingware.WingIDE101.installer.yaml:10]:
```suggestion
  InstallerSha256: 2875732AF1CBFE98E65E625215F82275E825DA001B50EF3F28620F4217A376FF
  AppsAndFeaturesEntries:
  - DisplayName: Wing 101 9.0.0.4
    ProductCode: Wing 101 9_is1
    Publisher: 
  ProductCode: Wing 101 9_is1
```

File: manifests/w/Wingware/WingIDE101/9.0.0.4/Wingware.WingIDE101.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Wingware.WingIDE101
+PackageVersion: 9.0.0.4
```
Comment (Trenly) [inline manifests/w/Wingware/WingIDE101/9.0.0.4/Wingware.WingIDE101.installer.yaml:5]:
```suggestion
PackageVersion: 9.0.0.4 # Null
```

File: manifests/w/Wingware/WingIDE101/9.0.0.4/Wingware.WingIDE101.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Wingware.WingIDE101
+PackageVersion: 9.0.0.4
+PackageLocale: en-US
+Publisher: Wingware
```
Comment (Trenly) [inline manifests/w/Wingware/WingIDE101/9.0.0.4/Wingware.WingIDE101.locale.en-US.yaml:7]:
```suggestion
Publisher: Wingware # Null
```

File: manifests/w/Wingware/WingIDE101/9.0.0.4/Wingware.WingIDE101.locale.en-US.yaml (hunk 1 above)
Comment (Trenly) [inline manifests/w/Wingware/WingIDE101/9.0.0.4/Wingware.WingIDE101.locale.en-US.yaml:11]:
```suggestion
ShortDescription: Wing 101 is the simplest IDE to teach students Python
```

File: manifests/w/Wingware/WingIDE101/9.0.0.4/Wingware.WingIDE101.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Wingware.WingIDE101
+PackageVersion: 9.0.0.4
+PackageLocale: en-US
+Publisher: Wingware
+PackageName: WingIDE 101
```
Comment (Trenly) [inline manifests/w/Wingware/WingIDE101/9.0.0.4/Wingware.WingIDE101.locale.en-US.yaml:8]:
```suggestion
PackageName: Wing 101
```
This should match Control Panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Trenly — APPROVED** [review]

**Pull request merged**